src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.Shared.xml

```diff
@@ -64,6 +64,7 @@
     <type fullname="System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder`1">
       <property name="ObjectIdForDebugger" />
     </type>
+    <type fullname="System.Runtime.CompilerServices.Unsafe" preserve="all" />
```

Comment:
This does not look right. Why is it needed?

Reply:
See: https://github.com/dotnet/runtime/pull/64861#issuecomment-1032930629